widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -4632,7 +4564,7 @@ div.dijitDateTextBox.dijitComboBox {
 }
 
 .fileBrowserColumn .body .file.selected {
-  background: rgba(204, 0, 0, 0.2);
+  background-color: rgba(204, 0, 0, 0.2) !important;
```

Comment:
Is this `!important` really needed?

Reply:
Looks like we can remove !important. Addressed.